protos/flyteidl/admin/task_execution.proto

```diff
@@ -101,6 +102,9 @@ message TaskExecutionClosure {
 message TaskExecutionGetDataRequest {
     // The identifier of the task execution for which to fetch inputs and outputs.
     core.TaskExecutionIdentifier id        = 1;
+
+    // Specifies what type of data to return in the response.
+    GetExecutionData.Mode mode             = 2;
```

Comment:
what is the default mode? Should we make the default as returning results inline?

Reply:
I made the default mode url blobs only to keep with existing behavior.